doc/api/errors.md

```diff
@@ -1185,6 +1185,11 @@ after the session had already closed.
 
 An error occurred while issuing a command via the `inspector` module.
 
+### ERR_INVALID_DEFINE
+
+An error occurred while setting a invalid attribute on the property of
```

Comment:
typo:

```suggestion
An error occurred while setting an invalid attribute on the property of
```

That being said, maybe it might make sense to name the error `ERR_INVALID_OBJECT_DEFINE_PROPERTY`, to maybe make it more obvious what this error is about.